From: Andreas Rheinhardt <andreas.rheinhardt@outlook.com>
Date: Mon, 1 Aug 2022 14:24:51 +0200
Subject: tests/fate-run: Allow to skip file checksums for lavf_image

The output file (even the filesize) of the recently added
EXR tests depends on the endianness; therefore checksums
of these files must not be part of the ref file. Therefore
this commit adds an option (unused for now) to disable these
checksums on a per-test basis.

In order to avoid having to check twice, the checksum and
the filesize info are moved to immediately follow one another;
this results into updates to the ref files of all lavf-image tests.

Signed-off-by: Andreas Rheinhardt <andreas.rheinhardt@outlook.com>
(cherry picked from commit 4fb8741c4670965eaf1f78d6122c6bdfdf1f3358)
---
 tests/fate-run.sh                   | 7 +++++--
 tests/ref/lavf/bmp                  | 2 +-
 tests/ref/lavf/dpx                  | 2 +-
 tests/ref/lavf/gbrp10le.dpx         | 2 +-
 tests/ref/lavf/gbrp12le.dpx         | 2 +-
 tests/ref/lavf/gbrpf32be.pfm        | 2 +-
 tests/ref/lavf/gbrpf32le.pfm        | 2 +-
 tests/ref/lavf/gray.pam             | 2 +-
 tests/ref/lavf/gray.xwd             | 2 +-
 tests/ref/lavf/gray16be.pam         | 2 +-
 tests/ref/lavf/gray16be.png         | 2 +-
 tests/ref/lavf/grayf32be.pfm        | 2 +-
 tests/ref/lavf/grayf32le.pfm        | 2 +-
 tests/ref/lavf/jpg                  | 2 +-
 tests/ref/lavf/monob.pam            | 2 +-
 tests/ref/lavf/monow.xwd            | 2 +-
 tests/ref/lavf/none.gbrapf32le.exr  | 2 +-
 tests/ref/lavf/none.gbrpf32le.exr   | 2 +-
 tests/ref/lavf/none.grayf32le.exr   | 2 +-
 tests/ref/lavf/pam                  | 2 +-
 tests/ref/lavf/pcx                  | 2 +-
 tests/ref/lavf/pgm                  | 2 +-
 tests/ref/lavf/png                  | 2 +-
 tests/ref/lavf/ppm                  | 2 +-
 tests/ref/lavf/qoi                  | 2 +-
 tests/ref/lavf/rgb48be.pam          | 2 +-
 tests/ref/lavf/rgb48be.png          | 2 +-
 tests/ref/lavf/rgb48le.dpx          | 2 +-
 tests/ref/lavf/rgb48le_10.dpx       | 2 +-
 tests/ref/lavf/rgb4_byte.xwd        | 2 +-
 tests/ref/lavf/rgb555be.xwd         | 2 +-
 tests/ref/lavf/rgb565be.xwd         | 2 +-
 tests/ref/lavf/rgb8.xwd             | 2 +-
 tests/ref/lavf/rgba.pam             | 2 +-
 tests/ref/lavf/rgba.xwd             | 2 +-
 tests/ref/lavf/rgba64le.dpx         | 2 +-
 tests/ref/lavf/rle.gbrapf32le.exr   | 2 +-
 tests/ref/lavf/rle.gbrpf32le.exr    | 2 +-
 tests/ref/lavf/rle.grayf32le.exr    | 2 +-
 tests/ref/lavf/sgi                  | 2 +-
 tests/ref/lavf/sun                  | 2 +-
 tests/ref/lavf/tga                  | 2 +-
 tests/ref/lavf/tiff                 | 2 +-
 tests/ref/lavf/xbm                  | 2 +-
 tests/ref/lavf/xwd                  | 2 +-
 tests/ref/lavf/zip1.gbrapf32le.exr  | 2 +-
 tests/ref/lavf/zip1.gbrpf32le.exr   | 2 +-
 tests/ref/lavf/zip1.grayf32le.exr   | 2 +-
 tests/ref/lavf/zip16.gbrapf32le.exr | 2 +-
 tests/ref/lavf/zip16.gbrpf32le.exr  | 2 +-
 tests/ref/lavf/zip16.grayf32le.exr  | 2 +-
 51 files changed, 55 insertions(+), 52 deletions(-)

diff --git a/tests/fate-run.sh b/tests/fate-run.sh
index 525e8e5..4008bcb 100755
--- a/tests/fate-run.sh
+++ b/tests/fate-run.sh
@@ -362,6 +362,7 @@ lavf_container_fate()
 }
 
 lavf_image(){
+    no_file_checksums="$3"
     nb_frames=13
     t="${test#lavf-}"
     outdir="tests/data/images/$t"
@@ -374,9 +375,11 @@ lavf_image(){
         done
     fi
     run_avconv $DEC_OPTS -f image2 -c:v pgmyuv -i $raw_src $1 "$ENC_OPTS -metadata title=lavftest" -vf scale -frames $nb_frames -y -qscale 10 $target_path/$file
-    do_md5sum ${outdir}/02.$t
+    if [ -z "$no_file_checksums" ]; then
+        do_md5sum ${outdir}/02.$t
+        echo $(wc -c ${outdir}/02.$t)
+    fi
     do_avconv_crc $file -auto_conversion_filters $DEC_OPTS $2 -i $target_path/$file $2
-    echo $(wc -c ${outdir}/02.$t)
 }
 
 lavf_image2pipe(){
diff --git a/tests/ref/lavf/bmp b/tests/ref/lavf/bmp
index af767e5..c4bc033 100644
--- a/tests/ref/lavf/bmp
+++ b/tests/ref/lavf/bmp
@@ -1,3 +1,3 @@
 71f4d64a6b3c71f43a4eff526f84841c *tests/data/images/bmp/02.bmp
-tests/data/images/bmp/%02d.bmp CRC=0xe6c71946
 304182 tests/data/images/bmp/02.bmp
+tests/data/images/bmp/%02d.bmp CRC=0xe6c71946
diff --git a/tests/ref/lavf/dpx b/tests/ref/lavf/dpx
index 68fe25a..736fe39 100644
--- a/tests/ref/lavf/dpx
+++ b/tests/ref/lavf/dpx
@@ -1,3 +1,3 @@
 4c8880d5835ffb5fe37c1ed8c8d404de *tests/data/images/dpx/02.dpx
-tests/data/images/dpx/%02d.dpx CRC=0x6da01946
 305792 tests/data/images/dpx/02.dpx
+tests/data/images/dpx/%02d.dpx CRC=0x6da01946
diff --git a/tests/ref/lavf/gbrp10le.dpx b/tests/ref/lavf/gbrp10le.dpx
index b33da34..35e34d0 100644
--- a/tests/ref/lavf/gbrp10le.dpx
+++ b/tests/ref/lavf/gbrp10le.dpx
@@ -1,3 +1,3 @@
 7ca935d5d5e00c54acbc85565d3039b6 *tests/data/images/gbrp10le.dpx/02.gbrp10le.dpx
-tests/data/images/gbrp10le.dpx/%02d.gbrp10le.dpx CRC=0xe6663fba
 407168 tests/data/images/gbrp10le.dpx/02.gbrp10le.dpx
+tests/data/images/gbrp10le.dpx/%02d.gbrp10le.dpx CRC=0xe6663fba
diff --git a/tests/ref/lavf/gbrp12le.dpx b/tests/ref/lavf/gbrp12le.dpx
index e2e794e..2758982 100644
--- a/tests/ref/lavf/gbrp12le.dpx
+++ b/tests/ref/lavf/gbrp12le.dpx
@@ -1,3 +1,3 @@
 a4cfea1797c928f2eff73573e559675d *tests/data/images/gbrp12le.dpx/02.gbrp12le.dpx
-tests/data/images/gbrp12le.dpx/%02d.gbrp12le.dpx CRC=0x1c755633
 609920 tests/data/images/gbrp12le.dpx/02.gbrp12le.dpx
+tests/data/images/gbrp12le.dpx/%02d.gbrp12le.dpx CRC=0x1c755633
diff --git a/tests/ref/lavf/gbrpf32be.pfm b/tests/ref/lavf/gbrpf32be.pfm
index 2cd6354..ca5e1b1 100644
--- a/tests/ref/lavf/gbrpf32be.pfm
+++ b/tests/ref/lavf/gbrpf32be.pfm
@@ -1,3 +1,3 @@
 6d470f8d6018b95b45afafc14b7d161a *tests/data/images/gbrpf32be.pfm/02.gbrpf32be.pfm
-tests/data/images/gbrpf32be.pfm/%02d.gbrpf32be.pfm CRC=0x4b73053f
 1216532 tests/data/images/gbrpf32be.pfm/02.gbrpf32be.pfm
+tests/data/images/gbrpf32be.pfm/%02d.gbrpf32be.pfm CRC=0x4b73053f
diff --git a/tests/ref/lavf/gbrpf32le.pfm b/tests/ref/lavf/gbrpf32le.pfm
index 3ddd100..b3947a9 100644
--- a/tests/ref/lavf/gbrpf32le.pfm
+++ b/tests/ref/lavf/gbrpf32le.pfm
@@ -1,3 +1,3 @@
 892c5a05e1cbb3d2f7761d51e18b9c4c *tests/data/images/gbrpf32le.pfm/02.gbrpf32le.pfm
-tests/data/images/gbrpf32le.pfm/%02d.gbrpf32le.pfm CRC=0x95e1053f
 1216533 tests/data/images/gbrpf32le.pfm/02.gbrpf32le.pfm
+tests/data/images/gbrpf32le.pfm/%02d.gbrpf32le.pfm CRC=0x95e1053f
diff --git a/tests/ref/lavf/gray.pam b/tests/ref/lavf/gray.pam
index 77af981..9283413 100644
--- a/tests/ref/lavf/gray.pam
+++ b/tests/ref/lavf/gray.pam
@@ -1,3 +1,3 @@
 35cb9e42b2d3181be494f8693af1ddea *tests/data/images/gray.pam/02.gray.pam
-tests/data/images/gray.pam/%02d.gray.pam CRC=0x0ff205be
 101445 tests/data/images/gray.pam/02.gray.pam
+tests/data/images/gray.pam/%02d.gray.pam CRC=0x0ff205be
diff --git a/tests/ref/lavf/gray.xwd b/tests/ref/lavf/gray.xwd
index 15c80fb..38b7f24 100644
--- a/tests/ref/lavf/gray.xwd
+++ b/tests/ref/lavf/gray.xwd
@@ -1,3 +1,3 @@
 85e9b8b814a1dea71d143aac2e487037 *tests/data/images/gray.xwd/02.gray.xwd
-tests/data/images/gray.xwd/%02d.gray.xwd CRC=0x0ff205be
 101487 tests/data/images/gray.xwd/02.gray.xwd
+tests/data/images/gray.xwd/%02d.gray.xwd CRC=0x0ff205be
diff --git a/tests/ref/lavf/gray16be.pam b/tests/ref/lavf/gray16be.pam
index 5038384..1182b83 100644
--- a/tests/ref/lavf/gray16be.pam
+++ b/tests/ref/lavf/gray16be.pam
@@ -1,3 +1,3 @@
 740eb42157af9e9eed46b70ba6a6cf4d *tests/data/images/gray16be.pam/02.gray16be.pam
-tests/data/images/gray16be.pam/%02d.gray16be.pam CRC=0x893f10ef
 202823 tests/data/images/gray16be.pam/02.gray16be.pam
+tests/data/images/gray16be.pam/%02d.gray16be.pam CRC=0x893f10ef
diff --git a/tests/ref/lavf/gray16be.png b/tests/ref/lavf/gray16be.png
index 2f52a83..4f4ce17 100644
--- a/tests/ref/lavf/gray16be.png
+++ b/tests/ref/lavf/gray16be.png
@@ -1,3 +1,3 @@
 6cf54c13aa407b77547cf6dfe23ecba3 *tests/data/images/gray16be.png/02.gray16be.png
-tests/data/images/gray16be.png/%02d.gray16be.png CRC=0x893f10ef
 47365 tests/data/images/gray16be.png/02.gray16be.png
+tests/data/images/gray16be.png/%02d.gray16be.png CRC=0x893f10ef
diff --git a/tests/ref/lavf/grayf32be.pfm b/tests/ref/lavf/grayf32be.pfm
index 05614a0..19a2ca8 100644
--- a/tests/ref/lavf/grayf32be.pfm
+++ b/tests/ref/lavf/grayf32be.pfm
@@ -1,3 +1,3 @@
 0f6df0d68d7dd30e67386b1255f443c9 *tests/data/images/grayf32be.pfm/02.grayf32be.pfm
-tests/data/images/grayf32be.pfm/%02d.grayf32be.pfm CRC=0xe3fda443
 405524 tests/data/images/grayf32be.pfm/02.grayf32be.pfm
+tests/data/images/grayf32be.pfm/%02d.grayf32be.pfm CRC=0xe3fda443
diff --git a/tests/ref/lavf/grayf32le.pfm b/tests/ref/lavf/grayf32le.pfm
index a419daa..aba861e 100644
--- a/tests/ref/lavf/grayf32le.pfm
+++ b/tests/ref/lavf/grayf32le.pfm
@@ -1,3 +1,3 @@
 145715872a894b1fde0105d8a0106191 *tests/data/images/grayf32le.pfm/02.grayf32le.pfm
-tests/data/images/grayf32le.pfm/%02d.grayf32le.pfm CRC=0x5443a443
 405525 tests/data/images/grayf32le.pfm/02.grayf32le.pfm
+tests/data/images/grayf32le.pfm/%02d.grayf32le.pfm CRC=0x5443a443
diff --git a/tests/ref/lavf/jpg b/tests/ref/lavf/jpg
index ecbb22c..94cee61 100644
--- a/tests/ref/lavf/jpg
+++ b/tests/ref/lavf/jpg
@@ -1,3 +1,3 @@
 1e7c6d937f21c045e0b238a83f62f3c5 *tests/data/images/jpg/02.jpg
-tests/data/images/jpg/%02d.jpg CRC=0xe3509f33
 26037 tests/data/images/jpg/02.jpg
+tests/data/images/jpg/%02d.jpg CRC=0xe3509f33
diff --git a/tests/ref/lavf/monob.pam b/tests/ref/lavf/monob.pam
index 488907e..a81be2c 100644
--- a/tests/ref/lavf/monob.pam
+++ b/tests/ref/lavf/monob.pam
@@ -1,3 +1,3 @@
 d2f5eb2f959ca3a90c02f1887b6e0c4f *tests/data/images/monob.pam/02.monob.pam
-tests/data/images/monob.pam/%02d.monob.pam CRC=0xab19200d
 101447 tests/data/images/monob.pam/02.monob.pam
+tests/data/images/monob.pam/%02d.monob.pam CRC=0xab19200d
diff --git a/tests/ref/lavf/monow.xwd b/tests/ref/lavf/monow.xwd
index da09d8b..10cd05c 100644
--- a/tests/ref/lavf/monow.xwd
+++ b/tests/ref/lavf/monow.xwd
@@ -1,3 +1,3 @@
 796e2e309ac0844cfb2f4959816508ee *tests/data/images/monow.xwd/02.monow.xwd
-tests/data/images/monow.xwd/%02d.monow.xwd CRC=0xc9a20204
 12783 tests/data/images/monow.xwd/02.monow.xwd
+tests/data/images/monow.xwd/%02d.monow.xwd CRC=0xc9a20204
diff --git a/tests/ref/lavf/none.gbrapf32le.exr b/tests/ref/lavf/none.gbrapf32le.exr
index 3250be2..099f36c 100644
--- a/tests/ref/lavf/none.gbrapf32le.exr
+++ b/tests/ref/lavf/none.gbrapf32le.exr
@@ -1,3 +1,3 @@
 c586035e67f9ba7f2a3777933b5b22a0 *tests/data/images/none.gbrapf32le.exr/02.none.gbrapf32le.exr
-tests/data/images/none.gbrapf32le.exr/%02d.none.gbrapf32le.exr CRC=0x068aca4e
 1627003 tests/data/images/none.gbrapf32le.exr/02.none.gbrapf32le.exr
+tests/data/images/none.gbrapf32le.exr/%02d.none.gbrapf32le.exr CRC=0x068aca4e
diff --git a/tests/ref/lavf/none.gbrpf32le.exr b/tests/ref/lavf/none.gbrpf32le.exr
index 41c43e3..7e0b2bd 100644
--- a/tests/ref/lavf/none.gbrpf32le.exr
+++ b/tests/ref/lavf/none.gbrpf32le.exr
@@ -1,3 +1,3 @@
 10f42423c6585fe5053c5457fba7b235 *tests/data/images/none.gbrpf32le.exr/02.none.gbrpf32le.exr
-tests/data/images/none.gbrpf32le.exr/%02d.none.gbrpf32le.exr CRC=0x95e1053f
 1221481 tests/data/images/none.gbrpf32le.exr/02.none.gbrpf32le.exr
+tests/data/images/none.gbrpf32le.exr/%02d.none.gbrpf32le.exr CRC=0x95e1053f
diff --git a/tests/ref/lavf/none.grayf32le.exr b/tests/ref/lavf/none.grayf32le.exr
index 19ab9a2..4d5fad8 100644
--- a/tests/ref/lavf/none.grayf32le.exr
+++ b/tests/ref/lavf/none.grayf32le.exr
@@ -1,3 +1,3 @@
 8aa28b10bf2591b7030b78bc29907293 *tests/data/images/none.grayf32le.exr/02.none.grayf32le.exr
-tests/data/images/none.grayf32le.exr/%02d.none.grayf32le.exr CRC=0x5443a443
 410437 tests/data/images/none.grayf32le.exr/02.none.grayf32le.exr
+tests/data/images/none.grayf32le.exr/%02d.none.grayf32le.exr CRC=0x5443a443
diff --git a/tests/ref/lavf/pam b/tests/ref/lavf/pam
index 8ac3f2b..ec3034d 100644
--- a/tests/ref/lavf/pam
+++ b/tests/ref/lavf/pam
@@ -1,3 +1,3 @@
 0dce5565222cf0f8b309467f279aecd2 *tests/data/images/pam/02.pam
-tests/data/images/pam/%02d.pam CRC=0x6da01946
 304191 tests/data/images/pam/02.pam
+tests/data/images/pam/%02d.pam CRC=0x6da01946
diff --git a/tests/ref/lavf/pcx b/tests/ref/lavf/pcx
index 2767ccf..8aa6eb4 100644
--- a/tests/ref/lavf/pcx
+++ b/tests/ref/lavf/pcx
@@ -1,3 +1,3 @@
 c4faf65ecc812ec8412cc26140c13bd5 *tests/data/images/pcx/02.pcx
-tests/data/images/pcx/%02d.pcx CRC=0x6da01946
 364147 tests/data/images/pcx/02.pcx
+tests/data/images/pcx/%02d.pcx CRC=0x6da01946
diff --git a/tests/ref/lavf/pgm b/tests/ref/lavf/pgm
index 4100ac9..814ecdc 100644
--- a/tests/ref/lavf/pgm
+++ b/tests/ref/lavf/pgm
@@ -1,3 +1,3 @@
 cc777c5fc4d116d4c5a996eac8d3133e *tests/data/images/pgm/02.pgm
-tests/data/images/pgm/%02d.pgm CRC=0x0ff205be
 101391 tests/data/images/pgm/02.pgm
+tests/data/images/pgm/%02d.pgm CRC=0x0ff205be
diff --git a/tests/ref/lavf/png b/tests/ref/lavf/png
index 165513e..2385ad1 100644
--- a/tests/ref/lavf/png
+++ b/tests/ref/lavf/png
@@ -1,3 +1,3 @@
 2af72da4468e61a37c220b25cb28618a *tests/data/images/png/02.png
-tests/data/images/png/%02d.png CRC=0x6da01946
 248633 tests/data/images/png/02.png
+tests/data/images/png/%02d.png CRC=0x6da01946
diff --git a/tests/ref/lavf/ppm b/tests/ref/lavf/ppm
index 04377f1..ce12659 100644
--- a/tests/ref/lavf/ppm
+++ b/tests/ref/lavf/ppm
@@ -1,3 +1,3 @@
 16d5dadf0b362fc8ba3cb676c5dde985 *tests/data/images/ppm/02.ppm
-tests/data/images/ppm/%02d.ppm CRC=0x6da01946
 304143 tests/data/images/ppm/02.ppm
+tests/data/images/ppm/%02d.ppm CRC=0x6da01946
diff --git a/tests/ref/lavf/qoi b/tests/ref/lavf/qoi
index 11cd832..76609ea 100644
--- a/tests/ref/lavf/qoi
+++ b/tests/ref/lavf/qoi
@@ -1,3 +1,3 @@
 3887a856d13f6444e213e9e268d58d55 *tests/data/images/qoi/02.qoi
-tests/data/images/qoi/%02d.qoi CRC=0x6da01946
 307658 tests/data/images/qoi/02.qoi
+tests/data/images/qoi/%02d.qoi CRC=0x6da01946
diff --git a/tests/ref/lavf/rgb48be.pam b/tests/ref/lavf/rgb48be.pam
index 13e5f77..7ad9f5c 100644
--- a/tests/ref/lavf/rgb48be.pam
+++ b/tests/ref/lavf/rgb48be.pam
@@ -1,3 +1,3 @@
 032538f0313b4f240b44a5bef115f5bf *tests/data/images/rgb48be.pam/02.rgb48be.pam
-tests/data/images/rgb48be.pam/%02d.rgb48be.pam CRC=0x5984c023
 608321 tests/data/images/rgb48be.pam/02.rgb48be.pam
+tests/data/images/rgb48be.pam/%02d.rgb48be.pam CRC=0x5984c023
diff --git a/tests/ref/lavf/rgb48be.png b/tests/ref/lavf/rgb48be.png
index 84f4d46..b893401 100644
--- a/tests/ref/lavf/rgb48be.png
+++ b/tests/ref/lavf/rgb48be.png
@@ -1,3 +1,3 @@
 b4e38244c97debe3f528e7d1adb283ef *tests/data/images/rgb48be.png/02.rgb48be.png
-tests/data/images/rgb48be.png/%02d.rgb48be.png CRC=0x5984c023
 511900 tests/data/images/rgb48be.png/02.rgb48be.png
+tests/data/images/rgb48be.png/%02d.rgb48be.png CRC=0x5984c023
diff --git a/tests/ref/lavf/rgb48le.dpx b/tests/ref/lavf/rgb48le.dpx
index 0731538..b808ff4 100644
--- a/tests/ref/lavf/rgb48le.dpx
+++ b/tests/ref/lavf/rgb48le.dpx
@@ -1,3 +1,3 @@
 075963c3c08978b6a20555ba09161434 *tests/data/images/rgb48le.dpx/02.rgb48le.dpx
-tests/data/images/rgb48le.dpx/%02d.rgb48le.dpx CRC=0xe5b9c023
 609920 tests/data/images/rgb48le.dpx/02.rgb48le.dpx
+tests/data/images/rgb48le.dpx/%02d.rgb48le.dpx CRC=0xe5b9c023
diff --git a/tests/ref/lavf/rgb48le_10.dpx b/tests/ref/lavf/rgb48le_10.dpx
index ce36e50..a49d929 100644
--- a/tests/ref/lavf/rgb48le_10.dpx
+++ b/tests/ref/lavf/rgb48le_10.dpx
@@ -1,3 +1,3 @@
 b9f22728f8ff393bf30cf6cbd624fa95 *tests/data/images/rgb48le_10.dpx/02.rgb48le_10.dpx
-tests/data/images/rgb48le_10.dpx/%02d.rgb48le_10.dpx CRC=0xf38d5830
 407168 tests/data/images/rgb48le_10.dpx/02.rgb48le_10.dpx
+tests/data/images/rgb48le_10.dpx/%02d.rgb48le_10.dpx CRC=0xf38d5830
diff --git a/tests/ref/lavf/rgb4_byte.xwd b/tests/ref/lavf/rgb4_byte.xwd
index fee4cf7..17546d1 100644
--- a/tests/ref/lavf/rgb4_byte.xwd
+++ b/tests/ref/lavf/rgb4_byte.xwd
@@ -1,3 +1,3 @@
 fe1af954966a40c2cd35fc27094ff823 *tests/data/images/rgb4_byte.xwd/02.rgb4_byte.xwd
-tests/data/images/rgb4_byte.xwd/%02d.rgb4_byte.xwd CRC=0xce042dcc
 104559 tests/data/images/rgb4_byte.xwd/02.rgb4_byte.xwd
+tests/data/images/rgb4_byte.xwd/%02d.rgb4_byte.xwd CRC=0xce042dcc
diff --git a/tests/ref/lavf/rgb555be.xwd b/tests/ref/lavf/rgb555be.xwd
index 1f9c579..53b81d5 100644
--- a/tests/ref/lavf/rgb555be.xwd
+++ b/tests/ref/lavf/rgb555be.xwd
@@ -1,3 +1,3 @@
 1300938325d5ac12caa09a43bd58f37c *tests/data/images/rgb555be.xwd/02.rgb555be.xwd
-tests/data/images/rgb555be.xwd/%02d.rgb555be.xwd CRC=0x14555d6e
 202863 tests/data/images/rgb555be.xwd/02.rgb555be.xwd
+tests/data/images/rgb555be.xwd/%02d.rgb555be.xwd CRC=0x14555d6e
diff --git a/tests/ref/lavf/rgb565be.xwd b/tests/ref/lavf/rgb565be.xwd
index 02a5bc2..1372fac 100644
--- a/tests/ref/lavf/rgb565be.xwd
+++ b/tests/ref/lavf/rgb565be.xwd
@@ -1,3 +1,3 @@
 c0866e9e710fce735423594a93bee604 *tests/data/images/rgb565be.xwd/02.rgb565be.xwd
-tests/data/images/rgb565be.xwd/%02d.rgb565be.xwd CRC=0x53209216
 202863 tests/data/images/rgb565be.xwd/02.rgb565be.xwd
+tests/data/images/rgb565be.xwd/%02d.rgb565be.xwd CRC=0x53209216
diff --git a/tests/ref/lavf/rgb8.xwd b/tests/ref/lavf/rgb8.xwd
index 52f4e3e..b19763f 100644
--- a/tests/ref/lavf/rgb8.xwd
+++ b/tests/ref/lavf/rgb8.xwd
@@ -1,3 +1,3 @@
 c6f3cb7c45f7238474a89d2ad61a1caf *tests/data/images/rgb8.xwd/02.rgb8.xwd
-tests/data/images/rgb8.xwd/%02d.rgb8.xwd CRC=0xf217a95e
 104559 tests/data/images/rgb8.xwd/02.rgb8.xwd
+tests/data/images/rgb8.xwd/%02d.rgb8.xwd CRC=0xf217a95e
diff --git a/tests/ref/lavf/rgba.pam b/tests/ref/lavf/rgba.pam
index 56e43a4..4e3bf8c 100644
--- a/tests/ref/lavf/rgba.pam
+++ b/tests/ref/lavf/rgba.pam
@@ -1,3 +1,3 @@
 2ed31ca8d8de560afb3e0fd7a873cde5 *tests/data/images/rgba.pam/02.rgba.pam
-tests/data/images/rgba.pam/%02d.rgba.pam CRC=0xf07d29cd
 405573 tests/data/images/rgba.pam/02.rgba.pam
+tests/data/images/rgba.pam/%02d.rgba.pam CRC=0xf07d29cd
diff --git a/tests/ref/lavf/rgba.xwd b/tests/ref/lavf/rgba.xwd
index 95aafdc..2d0f568 100644
--- a/tests/ref/lavf/rgba.xwd
+++ b/tests/ref/lavf/rgba.xwd
@@ -1,3 +1,3 @@
 1cdb43599c956dc8563f1e09fac5df00 *tests/data/images/rgba.xwd/02.rgba.xwd
-tests/data/images/rgba.xwd/%02d.rgba.xwd CRC=0xf07d29cd
 405615 tests/data/images/rgba.xwd/02.rgba.xwd
+tests/data/images/rgba.xwd/%02d.rgba.xwd CRC=0xf07d29cd
diff --git a/tests/ref/lavf/rgba64le.dpx b/tests/ref/lavf/rgba64le.dpx
index b4092c9..5ccde89 100644
--- a/tests/ref/lavf/rgba64le.dpx
+++ b/tests/ref/lavf/rgba64le.dpx
@@ -1,3 +1,3 @@
 545603630f30dec2768c8ae8d12eb8ea *tests/data/images/rgba64le.dpx/02.rgba64le.dpx
-tests/data/images/rgba64le.dpx/%02d.rgba64le.dpx CRC=0xe72ce131
 812672 tests/data/images/rgba64le.dpx/02.rgba64le.dpx
+tests/data/images/rgba64le.dpx/%02d.rgba64le.dpx CRC=0xe72ce131
diff --git a/tests/ref/lavf/rle.gbrapf32le.exr b/tests/ref/lavf/rle.gbrapf32le.exr
index e8093da..130103e 100644
--- a/tests/ref/lavf/rle.gbrapf32le.exr
+++ b/tests/ref/lavf/rle.gbrapf32le.exr
@@ -1,3 +1,3 @@
 94398a5ce98bb7b1b78b2b807306f6d7 *tests/data/images/rle.gbrapf32le.exr/02.rle.gbrapf32le.exr
-tests/data/images/rle.gbrapf32le.exr/%02d.rle.gbrapf32le.exr CRC=0x068aca4e
 1436849 tests/data/images/rle.gbrapf32le.exr/02.rle.gbrapf32le.exr
+tests/data/images/rle.gbrapf32le.exr/%02d.rle.gbrapf32le.exr CRC=0x068aca4e
diff --git a/tests/ref/lavf/rle.gbrpf32le.exr b/tests/ref/lavf/rle.gbrpf32le.exr
index beb8bf4..f16d710 100644
--- a/tests/ref/lavf/rle.gbrpf32le.exr
+++ b/tests/ref/lavf/rle.gbrpf32le.exr
@@ -1,3 +1,3 @@
 fed878e1f4391314c37088085942e572 *tests/data/images/rle.gbrpf32le.exr/02.rle.gbrpf32le.exr
-tests/data/images/rle.gbrpf32le.exr/%02d.rle.gbrpf32le.exr CRC=0x95e1053f
 1208298 tests/data/images/rle.gbrpf32le.exr/02.rle.gbrpf32le.exr
+tests/data/images/rle.gbrpf32le.exr/%02d.rle.gbrpf32le.exr CRC=0x95e1053f
diff --git a/tests/ref/lavf/rle.grayf32le.exr b/tests/ref/lavf/rle.grayf32le.exr
index 883b1d6..1ac6dfa 100644
--- a/tests/ref/lavf/rle.grayf32le.exr
+++ b/tests/ref/lavf/rle.grayf32le.exr
@@ -1,3 +1,3 @@
 2841f839cb5aa98bb7aded800dda7cc7 *tests/data/images/rle.grayf32le.exr/02.rle.grayf32le.exr
-tests/data/images/rle.grayf32le.exr/%02d.rle.grayf32le.exr CRC=0x5443a443
 410437 tests/data/images/rle.grayf32le.exr/02.rle.grayf32le.exr
+tests/data/images/rle.grayf32le.exr/%02d.rle.grayf32le.exr CRC=0x5443a443
diff --git a/tests/ref/lavf/sgi b/tests/ref/lavf/sgi
index 5049278..ad27b80 100644
--- a/tests/ref/lavf/sgi
+++ b/tests/ref/lavf/sgi
@@ -1,3 +1,3 @@
 d446e540a7c18da5fd3cc0e9942cd46f *tests/data/images/sgi/02.sgi
-tests/data/images/sgi/%02d.sgi CRC=0x6da01946
 307287 tests/data/images/sgi/02.sgi
+tests/data/images/sgi/%02d.sgi CRC=0x6da01946
diff --git a/tests/ref/lavf/sun b/tests/ref/lavf/sun
index 5dc0011..3aa76e4 100644
--- a/tests/ref/lavf/sun
+++ b/tests/ref/lavf/sun
@@ -1,3 +1,3 @@
 07518bcb0841bc677ce6aea8464ea240 *tests/data/images/sun/02.sun
-tests/data/images/sun/%02d.sun CRC=0xe6c71946
 304123 tests/data/images/sun/02.sun
+tests/data/images/sun/%02d.sun CRC=0xe6c71946
diff --git a/tests/ref/lavf/tga b/tests/ref/lavf/tga
index c7e33b0..96c5acd 100644
--- a/tests/ref/lavf/tga
+++ b/tests/ref/lavf/tga
@@ -1,3 +1,3 @@
 c0305c53e6d79d4ed9f35f04f671246c *tests/data/images/tga/02.tga
-tests/data/images/tga/%02d.tga CRC=0xe6c71946
 304172 tests/data/images/tga/02.tga
+tests/data/images/tga/%02d.tga CRC=0xe6c71946
diff --git a/tests/ref/lavf/tiff b/tests/ref/lavf/tiff
index c708642..35655b1 100644
--- a/tests/ref/lavf/tiff
+++ b/tests/ref/lavf/tiff
@@ -1,3 +1,3 @@
 b3299346a8959553a437e486d8f3bf76 *tests/data/images/tiff/02.tiff
-tests/data/images/tiff/%02d.tiff CRC=0x6da01946
 307131 tests/data/images/tiff/02.tiff
+tests/data/images/tiff/%02d.tiff CRC=0x6da01946
diff --git a/tests/ref/lavf/xbm b/tests/ref/lavf/xbm
index e54d6bc..c2a91f0 100644
--- a/tests/ref/lavf/xbm
+++ b/tests/ref/lavf/xbm
@@ -1,3 +1,3 @@
 83ed197cc88f382d9253365ffef70ec5 *tests/data/images/xbm/02.xbm
-tests/data/images/xbm/%02d.xbm CRC=0xc9a20204
 76410 tests/data/images/xbm/02.xbm
+tests/data/images/xbm/%02d.xbm CRC=0xc9a20204
diff --git a/tests/ref/lavf/xwd b/tests/ref/lavf/xwd
index 7a426ed..c51b3a9 100644
--- a/tests/ref/lavf/xwd
+++ b/tests/ref/lavf/xwd
@@ -1,3 +1,3 @@
 50baa5560b7d1aa3188b19c1162bf7dc *tests/data/images/xwd/02.xwd
-tests/data/images/xwd/%02d.xwd CRC=0x6da01946
 304239 tests/data/images/xwd/02.xwd
+tests/data/images/xwd/%02d.xwd CRC=0x6da01946
diff --git a/tests/ref/lavf/zip1.gbrapf32le.exr b/tests/ref/lavf/zip1.gbrapf32le.exr
index 7b2f21b..27d6b7b 100644
--- a/tests/ref/lavf/zip1.gbrapf32le.exr
+++ b/tests/ref/lavf/zip1.gbrapf32le.exr
@@ -1,3 +1,3 @@
 9fca73aac1a2e38969bed55929da48b4 *tests/data/images/zip1.gbrapf32le.exr/02.zip1.gbrapf32le.exr
-tests/data/images/zip1.gbrapf32le.exr/%02d.zip1.gbrapf32le.exr CRC=0x068aca4e
 902782 tests/data/images/zip1.gbrapf32le.exr/02.zip1.gbrapf32le.exr
+tests/data/images/zip1.gbrapf32le.exr/%02d.zip1.gbrapf32le.exr CRC=0x068aca4e
diff --git a/tests/ref/lavf/zip1.gbrpf32le.exr b/tests/ref/lavf/zip1.gbrpf32le.exr
index a986591..a85c786 100644
--- a/tests/ref/lavf/zip1.gbrpf32le.exr
+++ b/tests/ref/lavf/zip1.gbrpf32le.exr
@@ -1,3 +1,3 @@
 a405ee4328719c53f97d351b5910e82b *tests/data/images/zip1.gbrpf32le.exr/02.zip1.gbrpf32le.exr
-tests/data/images/zip1.gbrpf32le.exr/%02d.zip1.gbrpf32le.exr CRC=0x95e1053f
 897319 tests/data/images/zip1.gbrpf32le.exr/02.zip1.gbrpf32le.exr
+tests/data/images/zip1.gbrpf32le.exr/%02d.zip1.gbrpf32le.exr CRC=0x95e1053f
diff --git a/tests/ref/lavf/zip1.grayf32le.exr b/tests/ref/lavf/zip1.grayf32le.exr
index b89ceb2..536deb2 100644
--- a/tests/ref/lavf/zip1.grayf32le.exr
+++ b/tests/ref/lavf/zip1.grayf32le.exr
@@ -1,3 +1,3 @@
 76fe15b4e0b735a6318ca273ff52fb58 *tests/data/images/zip1.grayf32le.exr/02.zip1.grayf32le.exr
-tests/data/images/zip1.grayf32le.exr/%02d.zip1.grayf32le.exr CRC=0x5443a443
 91044 tests/data/images/zip1.grayf32le.exr/02.zip1.grayf32le.exr
+tests/data/images/zip1.grayf32le.exr/%02d.zip1.grayf32le.exr CRC=0x5443a443
diff --git a/tests/ref/lavf/zip16.gbrapf32le.exr b/tests/ref/lavf/zip16.gbrapf32le.exr
index 830ae0f..4dd9814 100644
--- a/tests/ref/lavf/zip16.gbrapf32le.exr
+++ b/tests/ref/lavf/zip16.gbrapf32le.exr
@@ -1,3 +1,3 @@
 8beb8db200e658e74fdd4ed4c4ca9214 *tests/data/images/zip16.gbrapf32le.exr/02.zip16.gbrapf32le.exr
-tests/data/images/zip16.gbrapf32le.exr/%02d.zip16.gbrapf32le.exr CRC=0x068aca4e
 807513 tests/data/images/zip16.gbrapf32le.exr/02.zip16.gbrapf32le.exr
+tests/data/images/zip16.gbrapf32le.exr/%02d.zip16.gbrapf32le.exr CRC=0x068aca4e
diff --git a/tests/ref/lavf/zip16.gbrpf32le.exr b/tests/ref/lavf/zip16.gbrpf32le.exr
index 27fa2a7..7e6f144 100644
--- a/tests/ref/lavf/zip16.gbrpf32le.exr
+++ b/tests/ref/lavf/zip16.gbrpf32le.exr
@@ -1,3 +1,3 @@
 4bc7e2640ea086b7fc31cbcd4fff19f5 *tests/data/images/zip16.gbrpf32le.exr/02.zip16.gbrpf32le.exr
-tests/data/images/zip16.gbrpf32le.exr/%02d.zip16.gbrpf32le.exr CRC=0x95e1053f
 796693 tests/data/images/zip16.gbrpf32le.exr/02.zip16.gbrpf32le.exr
+tests/data/images/zip16.gbrpf32le.exr/%02d.zip16.gbrpf32le.exr CRC=0x95e1053f
diff --git a/tests/ref/lavf/zip16.grayf32le.exr b/tests/ref/lavf/zip16.grayf32le.exr
index 24f8252..da74286 100644
--- a/tests/ref/lavf/zip16.grayf32le.exr
+++ b/tests/ref/lavf/zip16.grayf32le.exr
@@ -1,3 +1,3 @@
 5d5def6a4f16cef1122280d80c7c527d *tests/data/images/zip16.grayf32le.exr/02.zip16.grayf32le.exr
-tests/data/images/zip16.grayf32le.exr/%02d.zip16.grayf32le.exr CRC=0x5443a443
 63997 tests/data/images/zip16.grayf32le.exr/02.zip16.grayf32le.exr
+tests/data/images/zip16.grayf32le.exr/%02d.zip16.grayf32le.exr CRC=0x5443a443
